app/src/main/java/com/sakaimobile/development/sakaiclient20/ui/fragments/WebFragment.java

```diff
@@ -92,6 +94,22 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
 
         // Present the expected URL
         webView.loadUrl(URL);
+
+//        webView.getSettings().setJavaScriptEnabled(true);
```

Comment:
Was this for testing or something else? Looks like trying to remove the top bar for Sakai WebViews, but if it's not fleshed out yet don't include it in the PR.